requirements-nover.txt

```diff
@@ -2,7 +2,7 @@
 -e git+https://github.com/DevTable/container-cloud-config.git#egg=container-cloud-config
 -e git+https://github.com/DevTable/python-etcd.git@sslfix#egg=python-etcd
 -e git+https://github.com/NateFerrero/oauth2lib.git#egg=oauth2lib
--e git+https://github.com/coreos/mockldap.git@v0.1.x#egg=mockldap
+-e git+https://github.com/coreos/mockldap.git#egg=mockldap  # TODO: Latest commit should probably be tagged and pinned here.
```

Comment:
`-e git+https://github.com/quay/mockldap.git@4265554a3d89fe39bf05b18e91607bec3fcf215a#egg=mockldap`
I previously forked a more recent release of it and applied our patch on top of it. It's in `requirements.txt` but forgot to add it to this file.

Reply:
@kleesc Good catch

Reply:
@kleesc This is fixed now.